### PR TITLE
feat: Update Next versioning approach

### PR DESCRIPTION
**Pull request by fengsh27**

### CODE REVIEW

File: src-tauri/tauri.conf.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "withGlobalTauri": true
   },
   "package": {
-    "productName": "ChatGPT Next Web",
+    "productName": "BioChatter-Next",
     "version": "2.9.12"
```
Comment (slobentanzer) [inline src-tauri/tauri.conf.json:12]: why is this version still here? it conflicts with the 1.0.0 we define in `package.json`, no?

**Review by slobentanzer — COMMENTED** [review]
Thanks, just one question about the version noted in the tauri conf

File: src-tauri/tauri.conf.json (hunk 1 above)
Comment (fengsh27) [reply src-tauri/tauri.conf.json:12]: Yes, the version is still there, but we won't use this. do we need to delete this file?

Comment (slobentanzer) [reply src-tauri/tauri.conf.json:12]: I don't know what the file does. Do you know?

Comment (fengsh27) [reply src-tauri/tauri.conf.json:12]: This file is used to build web app to a desktop app [Tauri](https://tauri.app/)

Comment (slobentanzer) [reply src-tauri/tauri.conf.json:12]: So should we keep that? Does it work out of the box?

Comment (slobentanzer) [reply src-tauri/tauri.conf.json:12]: We can just replace 2.9.12 with 1.0.0 and have bumpversion update this file as well. If the Tauri build works, that is.

Comment (fengsh27) [reply src-tauri/tauri.conf.json]: @slobentanzer , I introduced standard-version to bump version. It requires:
1. Always using squash commits when merging PRs
2. Add types to commit message like `feat:`, `fix:` `refactor:`, `test:`, and `doc:`

Then after each PR, we can run `yarn run release` to bump version and merge to main.

Comment (slobentanzer) [reply src-tauri/tauri.conf.json]: I don't know that, but to be clear: we add the types to the squash message of the merge, right? Not to the individual ones in the PR itself.

Comment (fengsh27) [reply src-tauri/tauri.conf.json]: Yes, only need to add type to squash message of the PR. Let's try it to see if it works

Comment (slobentanzer) [reply src-tauri/tauri.conf.json]: OK :)

**Comment by slobentanzer** [comment]
I merged. However, I don't know how `yarn run release` works. `bumpversion` has patch, minor, major arguments. Can you link the docs?

**Comment by fengsh27** [comment]
Here is its repo: [standar-version](https://github.com/conventional-changelog/standard-version), and it uses [conventionalcommits.org](https://conventionalcommits.org/) for version management.


***How does this relate to SemVer?***
fix type commits should be translated to PATCH releases. feat type commits should be translated to MINOR releases. Commits with BREAKING CHANGE in the commits, regardless of type, should be translated to MAJOR releases.

**Pull request merged**